schema.sql

```diff
@@ -0,0 +1,11 @@
+CREATE DATABASE vet_clinic;
+
+CREATE TABLE animals (
+    id int,
```

Comment:
- [ ] **[OPTIONAL]** Good job so far 👏 👏 , However By generating IDs automatically in the database, you can be sure that each ID will be unique within the table, eliminating the possibility of duplicate IDs. This is especially important for primary key columns, as primary keys are used to uniquely identify each row in a table. 😎😎

so it would be better to add an auto-increment to your `id` like this: ```id int NOT NULL IDENTITY(1,1)``` 🙏